Module/Rule.Registry/Convert/Functions.SingleLine.ps1

```diff
@@ -196,6 +196,39 @@ function Get-RegistryValueTypeFromSingleLineStig
         }
     }
 }
+<#
+    .SYNOPSIS
+        Extract the registry path from an McaAfee STIG string.
+    .PARAMETER CheckContent
+        An array of the raw string data taken from the STIG setting.
+#>
+function Get-McAfeeRegistryPath
+{
+    [CmdletBinding()]
+    [OutputType([string])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [psobject]
+        $CheckContent
+    )
+
+    if ($checkContent -match "Software\\McAfee")
```

Comment:
$CheckContent should match the case from the Parameter, so on this line as well as line 221 and 225 should be "$CheckContent"